docs/rector_rules_overview.md

```diff
@@ -13557,105 +13455,6 @@ Remove php version checks if they are passed
 
 <br>
 
-## PostRector
-
-### ClassRenamingPostRector
-
-Post Rector that renames classes
-
-- class: `Rector\PostRector\Rector\ClassRenamingPostRector`
-
-```diff
--$someClass = new SomeClass();
-+$someClass = new AnotherClass();
-```
-
-<br>
-
-### NameImportingPostRector
-
-Imports fully qualified class names in parameter types, return types, extended classes, implemented, interfaces and even docblocks
-
-- class: `Rector\PostRector\Rector\NameImportingPostRector`
-
-```diff
--$someClass = new \Some\FullyQualified\SomeClass();
-+use Some\FullyQualified\SomeClass;
-+
-+$someClass = new SomeClass();
-```
-
-<br>
-
-### NodeAddingPostRector
-
-Post Rector that adds nodes
-
-- class: `Rector\PostRector\Rector\NodeAddingPostRector`
-
-```diff
-+$string = new String_(...);
- $value = 1000;
-```
-
-<br>
-
-### NodeRemovingPostRector
-
-PostRector that removes nodes
-
-- class: `Rector\PostRector\Rector\NodeRemovingPostRector`
-
-```diff
--$value = 1000;
--$string = new String_(...);
-+$value = 1000;
-```
-
-<br>
-
-### NodeToReplacePostRector
-
-Post Rector that replaces one nodes with another
-
-- class: `Rector\PostRector\Rector\NodeToReplacePostRector`
-
-```diff
--$string = new String_(...);
-+$value = 1000;
-```
-
-<br>
-
-### PropertyAddingPostRector
-
-Post Rector that adds properties
-
-- class: `Rector\PostRector\Rector\PropertyAddingPostRector`
-
-```diff
- class SomeClass
- {
-+    public $someProperty;
- }
-```
-
-<br>
-
-### UseAddingPostRector
-
-Post Rector that adds use statements
-
-- class: `Rector\PostRector\Rector\UseAddingPostRector`
-
-```diff
-+use App\SomeClass;
-+
- $someClass = new SomeClass();
-```
-
-<br>
```

Comment:
These were internal-only rules, that some people confused for normal ones (logically).
Now we can get rid of them in documentation :)